### DISCUSSION TITLE
How can I recover lotus-miner

### DISCUSSION THREAD

**Discussion by lbj2004032** [filecoin-project/lotus#8930] [Storage Provider] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
lotus-miner version 1.15.2+mainnet+git.518dc962e.dirty
```

### Describe the Bug

I deleted it by mistake Lotus miner with no backup (Lotus miner backup)

I execute lotus miner init --actor=xxx --owner=xxx. After initialization, I execute the following instructions:

lotus-miner sectors list

No blocks output

### Logging Information

```text
lotus-miner sectors list
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by rjan90** [comment]
Hey @lbj2004032! Do I understand it correctly that you have deleted the `.lotusminer` folder? And wondering if it´s possible to recover it without a backup?

**Reply by lbj2004032** [reply]
How to do it

**Comment by rjan90** [comment, marked as answer]
Hey! To fully recover a storage provider you will need `backup.cbor` with the storage providers metadata, and ideally the `config.toml`, and `storage.json` files. 

That said, if you are missing the backup.cbor file you should still be able to initialize your storage provider as long as you have access to its actor/owner/adresses/etc and be able to prove the sectors you have on-chain until they expire. But everything that needs the metadata to function will cause issues. You can try to recover it using: `lotus-miner init --no-local-storage --actor=xxx--sector-size=xxGiB --owner=xxx --worker=xxx`
